File: Python/Barang.py
class Barang():
    __ID = ""
    __nama = ""
    __brand = ""
    __harga = ""

    def __init__(self, ID="", nama="", brand="", harga=""):
        self.__ID = ID
        self.__nama = nama
        self.__brand = brand
        self.__harga = harga

    def getID(self):
        return self.__ID

    def getnama(self):
        return self.__nama
    
    def getbrand(self):
        return self.__brand
    
    def getharga(self):
        return self.__harga
    
    def setID(self, ID):
        self.__ID = ID

    def setnama(self, nama):
        self.__nama = nama

    def setbrand(self, brand):
        self.__brand = brand

    def setharga(self, harga):
        self.__harga = harga

File: PHP/Barang.php
<?php

class Barang{

    private $id = '';
    private $nama = '';
    private $brand = '';
    private $harga = '';

    public function __construct( $id = '', $nama = '', $brand = '', $harga = ''){
        $this->id = $id;
        $this->nama = $nama;
        $this->brand = $brand;
        $this->harga = $harga;
    }

    //setter

    public function setid($id){
        $this->id = $id;
    }

    public function setnama($nama){
        $this->nama = $nama;
    }

    public function setbrand($brand){
        $this->brand = $brand;
    }

    public function setharga($harga){
        $this->harga = $harga;
    }

    //getter

    public function getid(){
        return $this->id;
    }

    public function getnama(){
        return $this->nama;
    }

    public function getbrand(){
        return $this->brand;
    }

    public function getharga(){
        return $this->harga;
    }
}


?>